docs/deploy.md

```diff
@@ -135,3 +135,54 @@ frontend:
 | /<*>.md        | /<*>.md        | 200 (Rewrite) |
 | /<*>.png       | /<*>.png       | 200 (Rewrite) |
 | /<*>           | /index.html    | 200 (Rewrite) |        
+
+
+## Docker
+
+>There is a simple demo for building docsify in docker.
+
+?> tips: you can get all the files for this demo from [here](https://github.com/Koooooo-7/docsify-docker).
+
+- Preparing Docsify initial files  
+  You need prepare the initial files (generated by run `docsify init ./docs`) instead of making in container.
+
+  ```sh
+  .nojekyll
+  index.html
+  README.md
+  ```
+
+- Dockerfile
+
+  ```Dockerfile
+    FROM node:latest
+    MAINTAINER Koooooo-7(https://github.com/Koooooo-7)
```

Comment:
As these are generic instruction being added to docsify docs and the link to the repo is being removed (above), please remove the MAINTAINER information.